prov/efa/src/efa_domain.c

```diff
@@ -197,6 +197,8 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 		return -FI_EINVAL;
 	}
 
+	// Check the value of environment variable FI_HMEM_CUDA_ENABLE_XFER
```

Comment:
Use `/* */` for C

Reply:
// is valid for C comments since the C99 standard.   Does libfabric require only using C90 standard features?

Reply:
Since other comments are using `/* */`, it is better to use `/* */` for consistency.